addons/mail/static/tests/discuss_tests.js

```diff
@@ -423,6 +423,90 @@ QUnit.test('@ mention in channel', function (assert) {
     });
 });
 
+QUnit.test('@ mention in mailing channel', function (assert) {
+    assert.expect(10);
+    var done = assert.async();
+
+    var fetchListenersDef = $.Deferred();
+
+    this.data.initMessaging = {
+        channel_slots: {
+            channel_channel: [{
+                id: 1,
+                channel_type: "channel",
+                name: "general",
+                mass_mailing: true,
+            }],
+        },
+    };
+
+    createDiscuss({
+        id: 1,
+        context: {},
+        params: {},
+        data: this.data,
+        services: this.services,
+        mockRPC: function (route, args) {
+            if (args.method === 'channel_fetch_listeners') {
+                fetchListenersDef.resolve();
+                return $.when([
+                    {id: 1, name: 'Admin'},
+                ]);
+            }
+            return this._super.apply(this, arguments);
+        },
+        debug: true,
+    })
+    .then(function (discuss) {
+        var $general = discuss.$('.o_mail_discuss_sidebar')
+                        .find('.o_mail_discuss_item[data-thread-id=1]');
+        assert.strictEqual($general.length, 1,
+            "should have the channel item with id 1");
+        assert.strictEqual($general.attr('title'), 'general',
```

Comment:
isn't this assert already done in an other test? This is not really a requirement for your test